webapi/src/main/scala/org/knora/webapi/routing/RouteUtilV1.scala

```diff
@@ -127,17 +130,14 @@ object RouteUtilV1 {
     acceptStandoffLinksToClientIDs: Boolean,
     userProfile: UserADM,
     log: Logger
-  )(implicit runtime: Runtime[MessageRelay]): Future[TextWithStandoffTagsV2] =
-    UnsafeZioRun.runToFuture {
-      val request = GetMappingRequestV2(mappingIri, userProfile)
-      for {
-        mappingResponse <- MessageRelay.ask[GetMappingResponseV2](request)
-        textWithStandoffTagV1 <-
-          ZIO.attempt(
-            StandoffTagUtilV2.convertXMLtoStandoffTagV2(xml, mappingResponse, acceptStandoffLinksToClientIDs, log)
-          )
-      } yield textWithStandoffTagV1
-    }
+  ): ZIO[MessageRelay, Throwable, TextWithStandoffTagsV2] =
+    for {
+      mappingResponse <- MessageRelay.ask[GetMappingResponseV2](GetMappingRequestV2(mappingIri, userProfile))
+      textWithStandoffTagV1 <-
```

Comment:
```suggestion
      textWithStandoffTagV2 <-
```

Reply:
Not sure if V1 is on purpose or not.. (not that it matters a lot as it is only a variable name)